src/test/java/com/github/mc1arke/sonarqube/plugin/ce/pullrequest/markup/ParagraphTest.java

```diff
@@ -40,4 +40,8 @@ public void testIsValidChildValidChildImage() {
         assertTrue(new Paragraph().isValidChild(new Image("", "")));
     }
 
+    @Test
+    public void testIsValidChildValidChildLink() {
```

Comment:
Duplication in method name?

Reply:
same naming scheme as the other ones: testIsValidChildValidChildImage, testIsValidChildValidChildText